model/ftn/ww3_grid.ftn

```diff
@@ -6329,6 +6349,8 @@
 !/NL2               '     TEMP DEPTH ARRAY TOO SMALL, .LE. ',I8/)
 !/NL3 1004 FORMAT (/' *** WAVEWATCH-III ERROR IN READNL : '/          &
 !/NL3               '     TEMP QPARMS ARRAY TOO SMALL, .LE. ',I8/)
+ 1005 FORMAT (/' *** WAVEWATCH III ERROR IN READNL : '/          &
+               '     UNDETERMINED CALENDAR: CAL360 and NOLEAP'/)
```

Comment:
Comment:

'     UNDETERMINED CALENDAR: Select only one of CAL360 and NOLEAP'

might be a little more explicit?

Reply:
Agree, will be changed

Reply:
Happy with the new text for CALTYPE